tests/fixtures/modifiers.js

```diff
@@ -167,6 +167,21 @@ const modifiersFixtures = [
 		'expected': '([Aa](?:a))',
 		'expectedFlags': ''
 	},
+	{
+		'pattern': '\\p{Lowercase_Letter}(?-i:k)',
+		'flags': 'iu',
+		'options': { unicodePropertyEscapes: 'transform', modifiers: 'transform' },
+		'matches': ['ck', 'Ck', 'δk', 'Δk', '\u{118A8}k', '\u{118C8}k'],
+		'nonMatches': ['cK', 'CK', 'δK', 'ΔK', '\u{118A8}K', '\u{118C8}K', 'c\u212A', 'C\u212A'],
+		'expectedFlags': 'u'
```

Comment:
We did not test the transform result here because it is very likely that `\p{Lowercase_Letter}` will change per every Unicode versions. Here we use the `matches` and `nonMatches` tests to ensure the transform result is good.